Automattic/jetpack
# Instant Search: Improve product result format layout

#### Changes proposed in this Pull Request:
* Adds a 5-column layout to the product result format at large breakpoints.

<img width="1355" alt="Screen Shot 2021-03-24 at 5 48 48 PM" src="https://user-images.githubusercontent.com/4044428/112398200-3f1b4900-8cc9-11eb-93be-3faf7cb4cfd8.png">


* Adds a placeholder visual if the result lacks a corresponding image.

<img width="1355" alt="Screen Shot 2021-03-24 at 5 49 00 PM" src="https://user-images.githubusercontent.com/4044428/112398212-45a9c080-8cc9-11eb-82e6-d49f5ee8987e.png">

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
* Apply this change to your site with a Jetpack Search subscription.
* Perform a site search; append `result_format=product` query string to force the product result format layout. Ensure that the results look reasonable on all breakpoints (e.g. desktop, tablet, mobile).
